azure-pipelines.yml

```diff
@@ -93,6 +93,19 @@ jobs:
           MB_ML_VER: "2014"
           CONFIG_PATH: "config_ilp64.sh"
 
+        py_3.10_32:
+          AZURE_PYTHON_VERSION: "3.9"
+          MB_PYTHON_VERSION: "3.10"
+          PLAT: "i686"
+          MB_ML_VER: "2014"
+          ENV_VARS_PATH: "env_vars_32.sh"
+          DOCKER_TEST_IMAGE: "multibuild/xenial_{PLAT}"
```

Comment:
```suggestion
```

Reply:
The tests are currently run via [this line](https://dev.azure.com/numpy/numpy/_build/results?buildId=18919&view=logs&j=4f922444-fdfe-5dcf-b824-02f86439ef14&t=6c1ffda0-4c58-5358-a74f-05db16f3afea&l=131)
```
+ docker run --rm -e PYTHON_VERSION=3.7 -e MB_PYTHON_VERSION=3.7 \
    -e UNICODE_WIDTH=32 -e CONFIG_PATH= -e WHEEL_SDIR=wheelhouse \
    -e MANYLINUX_URL= -e 'TEST_DEPENDS=pytest hypothesis cffi pytz' \
    -v /home/vsts/work/1/s:/io multibuild/xenial_i686 /io/multibuild/docker_test_wrap.sh
```

which uses `multibuild/xenial_i686`. I was intending for it to use the manylinux image instead by removing the line.

Reply:
Without the line the build still worked, but there was no 3.10 for the test, the docker image defaults to the multibuild xenial version for testing. I just commented out the 32 bit and ARM64 builds. I assume both will get fixed at some point.

Reply:
> I assume both will get fixed at some point.

The arm64 image for focal now exists. On i686 we are stuck testing on the manylinux image since 32-bit images do not exist for current Ubuntu distributions.